Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -1791,6 +1791,15 @@
                     }
                 }
             },
+            "TopicColor": {
+                "type": "object",
+                "properties": {
+                    "standardColorIdentifier": {
+                        "type": "string",
+                        "enum": ["blue", "gray", "green", "orange", "purple", "red", "yellow"]
```

Comment:
Question: would a future custom color use a different property in the render json?

Reply:
I think so – I wanted to give us the flexibility for that if we decide it's needed. My current thinking is that we'll likely have some kind of `customColorReference` here instead of a standard identifier.